specs/monitoring_integration.adoc

```diff
@@ -0,0 +1,256 @@
+= Spec - Introducing a new component monitoring-integration
+
+A new component called monitoring-Integration is introduced which integrates Grafana
+with tendrl core.
+
+
+== Problem description
+
+The new monitoring-integration component would primarily focus on dynamic
+configuration of grafana, based on the topology changes in etcd. Some of these
+responsibilities would be:
+
+* Monitoring-integration should create default dashboards in grafana using predefined json.
+  The dashboard json contains details regarding panels, metrics and alerts.
+* Configuring grafana authentication and authorization (https://github.com/Tendrl/specifications/pull/205),
+  based on changes in the primary tendrl AA scheme.
+* Keeping track of the topology changes in etcd to configure the grafana dashboards
+  to ensure that the correct nodes are part of the respective cluster dashboards.
+  (Not planned yet, for 1st milestone grafana is configured using some default
+  configuration from monitoring-integration)
+
+
+== Use Cases
+
+For 1st milestone:
+
+* Monitoring-Integration will create default dashboards in grafana.
+
+For 2nd milestone:
+
+* Grafana alerting events should be notified via tendrl.
+* Monitoring-integration should be able to configure grafana based on topology
+  changes in etcd.
+
+For 3rd milestone:
+
+* User should be able access grafana dashboard via tendrl user interface using
+  tendrl authentication.
+
```

Comment:
When you planing to handle the updates?for ex updates to json files during yum update?
Also monitoring-integration as a service?